replay/models/ucb.py

```diff
@@ -0,0 +1,114 @@
+from typing import Optional
+
+import pandas as pd
+
+from pyspark.sql import DataFrame, Window
+from pyspark.sql import functions as sf
+
+from replay.models.base_rec import Recommender
+
+
+class UCB(Recommender):
+    """
+    Calculates upper confidence bound (UCB) for the confidence interval
+    of true fraction of positive ratings.
+
+    ``relevance`` must be converted to binary 0-1 form.
+
+    >>> import pandas as pd
+    >>> data_frame = pd.DataFrame({"user_idx": [1, 2], "item_idx": [1, 2], "relevance": [1, 1]})
```

Comment:
add rows with 0 relevance, add item with only zero relevance, add new item/user to predict to show it works with cold

Reply:
you are right